lib/chef/provider/package/yum/python_helper.rb

```diff
@@ -115,17 +115,70 @@ def options_params(options)
             end
           end
 
+          def is_arch?(arch)
+            arches = %w{alpha alphaev4 alphaev45 alphaev5 alphaev56 alphaev6 alphaev67 alphaev68 alphaev7 alphapca56 amd64 armv5tejl armv5tel armv6l armv7l athlon geode i386 i486 i586 i686 ia32e ia64 noarch ppc ppc64 ppc64iseries ppc64pseries s390 s390x sh3 sh4 sh4a sparc sparc64 sparc64v sparcv8 sparcv9 sparcv9v x86_64}
```

Comment:
1/2 these need to get shoved into https://github.com/chef/chef/blob/main/cspell.json#L16

Reply:
i'm trying to go the other way and disable spellchecking on that line, its useless to have it